packages/node/src/Session.spec.ts

```diff
@@ -316,6 +316,31 @@ describe("Session", () => {
         mySession.handleIncomingRedirect("https://some.url")
       ).resolves.not.toThrow();
     });
+
+    it("sets the appropriate information before calling the callback", async (done) => {
+      let hasBeenCalled = false;
+      const clientAuthentication = mockClientAuthentication();
+      clientAuthentication.handleIncomingRedirect = jest.fn(
+        async (_url: string) => {
+          return {
+            isLoggedIn: true,
+            sessionId: "a session ID",
+            webId: "https://some.webid#them",
+          };
+        }
+      );
+      const mySession = new Session({ clientAuthentication });
+      const myCallback = (): void => {
+        hasBeenCalled = true;
+        expect(mySession.info.webId).toBe("https://some.webid#them");
+        if (done) {
```

Comment:
Likewise about the mock function and `expect.assertions` - and is this `done()` call really necessary here?

Reply:
That's a good point. I initially assumed (when implementing the tests back on `solid-auth-fetcher`) that it would be necessary to do so to prevent trhe test from completing before the callback is called, but it appears that it is not the case. Fixed in 80a6e1d11ae1bff9b4437654f60b9193209820e0